### PR TITLE
Fixing shit.

### PR DESCRIPTION
**Pull request by DeadliestDesire**
Might work, might not. dont care.
Use this as suggestions.
Fixes food and water problems, leather idiocracy, glass retardation.

## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

File: dwarfs/code/modules/glassworking/sand.dm [hunk 1]
```diff
@@ -14,8 +14,8 @@
 	novariants = TRUE
 
 GLOBAL_LIST_INIT(glass_recipes, list(
-		new/datum/stack_recipe("Demijohn", /obj/structure/demijohn, 15, time=60 SECONDS),
-		// new/datum/stack_recipe("Glass Bottle", /obj/item/reagent_containers/glass/bottle, 5, time=30 SECONDS)
+		new/datum/stack_recipe("Demijohn", /obj/structure/demijohn, 10, time=60 SECONDS),
+		new/datum/stack_recipe("Glass Bottle", /obj/item/reagent_containers/glass/bottle, 2, time=15 SECONDS)
```
Comment (Valtosin) [inline dwarfs/code/modules/glassworking/sand.dm:18]: squeezed the glass in just bare hands and got a bottle, brutal dwarf magic

**Comment by BartDrown** [comment]
Learn how to create readable pull request or sneed.
Also, breaks balance for leather work (great idea to spawn 8 leather from 1 cat).
Thirst and hunger are currently not decreasing on click so changing var does not do anything. 
Only starvation and hydration tresholds might be worth looking up, but dwarfs rather don't like to be empty bellied so for me still it doesn't make sense.
And next time delete changelog description for god sake.

**Comment by daxsc** [comment]
No, so many changes that make little sense and no explanation at all. I will be doing balancing tweaks today.

**Comment by DeadliestDesire** [comment]
> Learn how to create readable pull request or sneed. Also, breaks balance for leather work (great idea to spawn 8 leather from 1 cat). Thirst and hunger are currently not decreasing on click so changing var does not do anything. Only starvation and hydration tresholds might be worth looking up, but dwarfs rather don't like to be empty bellied so for me still it doesn't make sense. And next time delete changelog description for god sake.

Nerd energy overdrive right there, I made comprehensive and detailed suggestion in discord, discord deleted it over word  "shit".
Leather is not a fucking hide. You are speaking utter nonsense. Also cats are finite last time I checked, 8 fucking FULL SCALE HIDES for a single satchel, are you sure about whatever you just said?

You and max might be alright coders, but certainly do not understand what stable beta is. Or what is gameplay loop is. Right now, half content is simply not accessable due to lack of opportunity to reach it. You guys live this delusion that just because you coded it, you know where to lead that.

File: dwarfs/code/modules/glassworking/sand.dm (hunk 1 above)
Comment (DeadliestDesire) [reply dwarfs/code/modules/glassworking/sand.dm:18]: > squeezed the glass in just bare hands and got a bottle, brutal dwarf magic

Ah yes, I completely forgot that this is triple AAA project with top tier polish and certainly not beta for people to try things, why dont we add every real life detail into the game, why arent dwarfs suffocate in a cave completely sealed?

**Comment by BartDrown** [comment]
Cats are infinite, so skill issue
It's not beta, just pre test to balance shit like this, sole purpouse of it is to make it more fun and balanced, so skill issue 2
Prove it wasn't stable, unless it's skill issue  n° 3.

I repeat cause you don't seem to get it:
Purpouse of this launch is to determine balance. Being mad that it's not balanced it's just dumb. 
Also, rebalancing still will need further examination, it's not like you type one increased variable and it will start to work. Leather drop is too small, but it shouldn't be 8.  This is just "I'm mad cause I couldn't have it, so I'll modify so I can have it" type of pull request. It's far from balanced.
Talk it through, not demand changes up from ass, cause it won't happen

**Pull request closed without merging**